addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -984,6 +1007,9 @@ export class Rtc extends Record {
         });
     }
 
+    /**
+     * @param {SessionInfoMap} changes
+     */
```

Comment:
Nitpick: personally I prefer 1 line than 3 lines for such comments:
```js
/** @param {SessionInfoMap} changes */
```
This works too, and less LOCs is better because you have to scroll less and more content is visible in viewport.